scripts/install

```diff
@@ -1,156 +1,196 @@
-
 # Change variables in Wordpress configuration
-sudo sed -i "s/__DB_USER__/$db_user/g" $final_path/wp-config.php
+sudo sed -i "s/__DB_USER__/$db_name/g" $final_path/wp-config.php
 sudo sed -i "s/__DB_PWD__/$db_pwd/g" $final_path/wp-config.php
 sudo sed -i "s/__DOMAIN__/$domain/g" $final_path/wp-config.php
-sudo sed -i "s@__PATH__@$path@g" $final_path/wp-config.php
-
+sudo sed -i "s@__PATH__@$path_url@g" $final_path/wp-config.php
 
 for i in 1 2 3 4 5 6 7 8
 do
-    j=$(dd if=/dev/urandom bs=1 count=200 2> /dev/null | tr -c -d '[A-Za-z0-9]' | sed -n 's/\(.\{40\}\).*/\1/p')
-    if [ "$j" = "" ];
-    then
-        # For obscure reasons, the loop is too fast at execution
-        sleep 1
-        j=$(dd if=/dev/urandom bs=1 count=200 2> /dev/null | tr -c -d '[A-Za-z0-9]' | sed -n 's/\(.\{40\}\).*/\1/p')
-    fi
-    sudo sed -i "s/KEY$i/$j/g" $final_path/wp-config.php
+	j=$(dd if=/dev/urandom bs=1 count=200 2> /dev/null | tr -c -d '[A-Za-z0-9]' | sed -n 's/\(.\{40\}\).*/\1/p')
+	if [ "$j" = "" ];
+	then
+		# For obscure reasons, the loop is too fast at execution
+		sleep 1
+		j=$(dd if=/dev/urandom bs=1 count=200 2> /dev/null | tr -c -d '[A-Za-z0-9]' | sed -n 's/\(.\{40\}\).*/\1/p')
+	fi
+	sudo sed -i "s/KEY$i/$j/g" $final_path/wp-config.php
 done
 
-# Set permissions to wordpress directory
-sudo chown -R www-data: $final_path
+#=================================================
+# SETTING UP WITH CURL
+#=================================================
+
+# Set right permissions for curl install
+sudo chown -R $app: $final_path
 
-# Donne un accès public pour curl
+# Rend la page d'install publique pour curl
 ynh_app_setting_set $app unprotected_uris "/"
-# Recharge la configuration Nginx
-sudo service nginx reload
-# Régénère la configuration de SSOwat
-sudo yunohost app ssowatconf
+sudo yunohost app ssowatconf	# Régénère la configuration de SSOwat
 
+# Reload Nginx
+sudo systemctl reload nginx
 
 # Wordpress installation
-echo -e "127.0.0.1 $domain #WORDPRESS" | sudo tee -a /etc/hosts
-sleep 1
-curl -kL -H "Host: $domain" --data "&weblog_title=YunoBlog&user_name=$admin_wordpress&admin_password=$db_pwd&admin_password2=$db_pwd&admin_email=$admin_wordpress@$domain&language=$language&Submit=Install+WordPress" https://localhost$path/wp-admin/install.php?step=2 > /dev/null 2>&1
+YNH_CURL "&weblog_title=YunoBlog&user_name=$admin_wordpress&admin_password=$db_pwd&admin_password2=$db_pwd&admin_email=$admin_wordpress@$domain&language=$language&Submit=Install+WordPress" "/wp-admin/install.php?step=2"
 
-echo -n "Please wait during Wordpress installation" >&2
+WARNING echo -n "Please wait during Wordpress installation"
 for i in `seq 1 300`
 do	# La boucle attend la fin de l'installation de wordpress Ou 5 minutes.
-	if mysql --debug-check -u $db_user -p$db_pwd $db_user -e "show tables" | grep -q "wp_options"; then
+	if ynh_mysql_connect_as $db_name $db_pwd $db_name -e "show tables" | grep -q "wp_options"; then
```

Comment:
Yes you're right, that works on my CI, but not on a VM.
With this correction, that seems really better.